.github/CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ Generally speaking questions are better suited in our resources below.
 
 - The official support server: https://discord.gg/UCXwPR7Pew
 - The Discord API server under #python_discord-py: https://discord.gg/discord-api
-- [The FAQ in the documentation](https://pycord.readthedocs.io/en/latest/faq.html)
+- [The FAQ in the documentation](https://docs.pycord.dev/en/master/faq.html)
```

Comment:
not really needed it redirects to the new page anyway

Reply:
still please use the new one.
I changed the config to use master per default.
But the old one would redirect to latest